packages/link/src/react/index.js

```diff
@@ -1,31 +1,39 @@
-import * as glamor from 'glamor'
+import { css } from 'glamor'
+import filterReactProps from '@pluralsight/ps-design-system-filter-react-props'
 import PropTypes from 'prop-types'
 import React from 'react'
 
-import css from '../css'
-import * as vars from '../vars'
+import stylesheet from '../css/index.js'
+import * as vars from '../vars/index.js'
 
 const style = props =>
-  glamor.css({
-    ':hover': css['.psds-link:hover'],
-    ...(({ appearance }) => css[`.psds-link--appearance-${appearance}`])(props)
+  css(
+    {
+      ':hover': stylesheet['.psds-link:hover']
+    },
+    stylesheet[`.psds-link--appearance-${props.appearance}`]
+  )
+
+const Link = React.forwardRef((props, forwardedRef) => {
```

Comment:
does this count as our react16 upgrade for Link? let's mark it off the list !

Reply:
yes, I'll mark it.